src/test/java/step3/CarsTest.java

```diff
@@ -0,0 +1,68 @@
+package step3;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import step3.model.Car;
+import step3.model.Cars;
+import step3.utils.RacingUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarsTest {
+
+    private static final int TEST_CAR_COUNT_VALUE = 3;
+    private static final int MAX_BOUND = 10;
+    private static int ZERO = 0;
+
+    private Cars TestCar;
+
+    private void init(int carCount) {
+        TestCar = Cars.generateCars(carCount);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 3, 5})
+    void 입력된_숫자만큼의_Car_생성(int count) {
+        init(count);
+        assertThat(TestCar.getCars().size()).isEqualTo(count);
```

Comment:
아래 링크를 읽어보면 좋을 것 같아요. 
https://prohannah.tistory.com/204

```suggestion
        assertThat(TestCar.getCarSize())
                .isEqualTo(count);
```